addons/addon-base-raas/packages/base-raas-services/lib/data-source/access-strategy/roles-only/filesystem-role-service.js

```diff
@@ -378,10 +371,8 @@ class FilesystemRoleService extends Service {
       return;
     }
 
-    if (usage.removed) {
-      // Delete the role from the data source account
-      await this.deprovisionRole(fsRoleEntity);
-    }
+    // Delete the role from the data source account
+    await this.deprovisionRole(fsRoleEntity);
```

Comment:
What is the behavior of calling this method if the policy or the role is already deleted? If it throws an error, we need to fix it.